Display transaction timestamps

dkackman opened the pull request:
related to: https://github.com/xch-dev/sage/issues/38

Show block timestamp, formatted to user locale and timezone, on the transaction table and page.

Rigidity commented:
Overall, this works pretty well. Though I have some comments on the new list layout:

1. I think the coin list is too compact now (in fact, I'm probably going to try and increase the padding a bit on the token list in the future too)
2. The checkboxes aren't centered
3. I think it should default to only showing _unspent_ coins, currently it shows all of them by default which is confusing to people I think
4. The pagination buttons being at the bottom and requiring scrolling to get to is somewhat annoying
5. The addresses page feels a bit unrefined, but that's mostly unrelated to this PR

<img width="1088" alt="Screenshot 2025-03-21 at 11 47 31 PM" src="https://github.com/user-attachments/assets/0bf15364-b317-4f72-a180-c531981fa78a" />

dkackman commented:
> Overall, this works pretty well. Though I have some comments on the new list layout:
> 
>     1. I think the coin list is too compact now (in fact, I'm probably going to try and increase the padding a bit on the token list in the future too)

Along with the `AddressList` (the reason it changed below) all the tables now share the same base configuration. I've just checked in a pixel increase in row height that will change token, transaction, coin and address lists. Lemme know if they all look good. Can still customize individually.

> 
>     2. The checkboxes aren't centered

fixed

> 
>     3. I think it should default to only showing _unspent_ coins, currently it shows all of them by default which is confusing to people I think

fixed

> 
>     4. The pagination buttons being at the bottom and requiring scrolling to get to is somewhat annoying

This didn't change did it? I'll play around some alternatives

> 
>     5. The addresses page feels a bit unrefined, but that's mostly unrelated to this PR

I can roll this back but I do think it makes sense for all tables to get styled together.

> 
>
